app/views/idv/doc_auth/welcome.html.erb

```diff
@@ -3,105 +3,91 @@
 <% step = 0 %>
 <% if flow_session[:error_message] %>
   <%= render 'shared/alert', {
-    type: 'error',
-    class: 'margin-bottom-4',
-    message: flow_session[:error_message],
+      type: 'error',
+      class: 'margin-bottom-4',
+      message: flow_session[:error_message],
   } %>
 <% end %>
 
```

Comment:
Minor: This might not be needed anymore, since we don't have a form or errors associated with this step anymore.